src/Microsoft.Content.Build.MerfToSDPForJava.DataContracts/Hierarchy/HierarchyChange.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace Microsoft.Content.Build.MerfToSDPForJava.DataContracts
```

Comment:
why do we need to "HierarchyChange" for please?

Reply:
updated